tests/Microsoft.Identity.Test.Unit.net45/CoreTests/InstanceTests/InstanceDiscoveryManagerTests.cs

```diff
@@ -151,12 +170,12 @@ public async Task AuthorityValidationFailure_IsRethrown_Async()
         public async Task NetworkProviderFailures_AreIgnored_Async()
         {
             // Arrange
-            _staticMetadataProvider = new StaticMetadataProvider();
-            _knownMetadataProvider.GetMetadata("some_env.com", Enumerable.Empty<string>()).Returns(_expectedResult);
+            _networkCacheMetadataProvider = new NetworkCacheMetadataProvider();
+            _knownMetadataProvider.GetMetadata("some_env.com", Enumerable.Empty<string>(), Arg.Any<ICoreLogger>()).Returns(_expectedResult);
```

Comment:
should "some_env.com" be a constant?

Reply:
In product code, I'm all for constants. In test code, I tend to use strings inline because of readability. Tests are very difficult to read anyway and they also use much more constants than product code. Being able to read a test from top to bottom seems more important to me than changing the value of a test constant, which happens very rarely.